mlops/kubeflow/robustness_evaluation_fgsm_pytorch/component.yaml

```diff
@@ -0,0 +1,63 @@
+# MIT License
+#
+# Copyright (C) IBM Corporation 2019
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+name: 'Adversarial Robustness Evaluation'
+description: |
+  Perform an evaluation of the adversarial robustness using the Fast Gradient Sign Method of the Adversarial Robustness Toolbox (ART).
```

Comment:
I suggest to add an extra line here with a comment/disclaimer that it is not recommended to evaluate adversarial robustness solely using the Fast Gradient Sign Method, with a reference to https://arxiv.org/abs/1902.06705 which provides widely accepted guidelines which combinations of attacks to apply for a thorough robustness assessment.

Reply:
Hi @mathsinn, I updated the description and readme with some extra comment on using the Fast Gradient Sign Method for robustness evaluation. Is there anything else you want me to add in the description?